ocsp/responder/multi_source.go

```diff
@@ -60,7 +60,17 @@ func (src *multiSource) Response(ctx context.Context, req *ocsp.Request) (*Respo
 	// from reaching the backend layer (Redis) and causing connections to be closed
 	// unnecessarily.
 	// https://blog.uptrace.dev/posts/go-context-timeout.html
-	secondaryChan := getResponse(context.Background(), src.secondary, req)
+	redisCtx := context.Background()
+	deadline, ok := ctx.Deadline()
+	if ok {
+		// We don't call the CancelFunc returnwed by WithDeadline because it
```

Comment:
```suggestion
		// We don't call the CancelFunc returned by WithDeadline because it
```

Reply:
Weird that lint catches some typos but didn't catch this one?